sis-local/example/README.md

```diff
@@ -0,0 +1,33 @@
+# Example usage of streamlit_in_snowflake
+
+1. Create a secrets.toml file in this repo, as described in the parent directory README
+2. Create and activate a python 3.8 virtual environment, such as with Conda (see below)
+3. Run the following
+
+```shell
+pip install -r requirements.txt
```

Comment:
```suggestion
pip install "streamlit_in_snowflake @ git+https://github.com/sfc-gh-jcarroll/sis-tricks.git@sis-local#subdirectory=sis-local"
```

Reply:
Yeah, I think installing from this repo makes sense, so they don't have to do any manual copying and pasting of any extra files. I like that.